backend/Squirrel.ConsoleApp.BL/Extensions/Parsing.cs

```diff
@@ -0,0 +1,19 @@
+﻿namespace Squirrel.ConsoleApp.BL.Extensions
+{
+    public static class Parsing
+    {
+        public static int? ParseNullableInt(string value)
+        {
+            if (string.IsNullOrEmpty(value))
+                return null;
+            return int.Parse(value);
+        }
+
+        public static bool? ParseNullableBool(string value)
+        {
+            if (string.IsNullOrEmpty(value))
+                return null;
+            return bool.Parse(value);
+        }
+    }
+}
```

Comment:
transform it into extension methods or move this class out of the `Squirrel.ConsoleApp.BL.Extensions` namespace